custom_components/tahoma/__init__.py

```diff
@@ -40,17 +44,30 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
         _LOGGER.exception(exception)
         return False
 
-    devices = await client.get_devices()
     scenes = await client.get_scenarios()
 
+    coordinator = TahomaDataUpdateCoordinator(
+        hass,
+        _LOGGER,
+        # Name of the data. For logging purposes.
+        name="Tahoma fetcher",
+        client=client,
+        devices=await client.get_devices(),
+        listener_id=await client.register_event_listener(),
+        update_interval=timedelta(seconds=5),
```

Comment:
Is every 5 seconds not a bit to fast? Currently we only poll for updates when a device changes. This is great and will allow us to update the state if something happens outside HA.

But is every 30 seconds not enough? And maybe poll every second when we expect an update from a component, until we have received the new device state.

Reply:
Yes 5 seconds was for testing, I will set it to 30 seconds. About the second remark, we can see this in a second PR. This one is already big enough.

Reply:
@tetienne @vlebourl maybe we should keep 5 seconds for now? Until we switch to an approach where we update once every x minutes and every second after an update. 

If we set it to 30 seconds, updates will still be slow.

Reply:
I'm afraid 5 seconds will overlead the HA, it's recommended to keep high. I was not able to find a component where such low poll interval.

About your suggestion to reduce it, when we just do an update using HA, we can see if we can schedule an update 5s later.